api/views.py

```diff
@@ -99,14 +98,11 @@ def post(self, request):
         user = User.objects.get(username=request.user)
         q = request.data.get('type')
 
-        if q == 'Active Post':
-            posts = Post.objects.filter(user=user, active=True).order_by('-timestamp')
-            serializer = self.serializer_class(posts, many=True).data
-            return Response(serializer, status=HTTP_200_OK)
-        if q == 'Pending Post':
+        if q == 'Active Post' or q == 'Pending Post':
```

Comment:
If the q is Active post then you need to show only active post and vice versa

Reply:
I changed it please review it. now if q if active post it will show active as true otherwise as false.

Reply:
Looks good to me 💯 , you can also refactor my other project e.g moviesfy

Reply:
okay sure :)